spec/board_spec.rb

```diff
@@ -24,17 +24,43 @@
     end
   end
 
-  describe '#valid_coordinate?' do
-    context 'when passed a valid coordinate' do
+  describe '#valid_coordinate?' do #checks if a given coordinate is valid
+    context 'when passed a valid coordinate' do #context is similar to describe and used to group examples. specific state/condition
       subject(:valid) { board.valid_coordinate?('A1') }
 
       it { is_expected.to be true }
     end
 
-    context 'when passed an invalid coordinate' do
+    context 'when passed an invalid coordinate' do 
       subject(:invalid) { board.valid_coordinate?('A5') }
 
       it { is_expected.to be false }
     end
   end
+
+  describe '#valid_placement?' do #checks if a ship can be placed at the given coordinates
```

Comment:
This looks like the type of method where we can take advantage of context in RSpec to make the test output a bit more readable